crf/src/main/java/org/sagebase/crf/step/CrfStartTaskStepLayout.java

```diff
@@ -180,4 +199,43 @@ public void onActivityFinished(int requestCode, int resultCode, Intent data) {
 
         }
     }
+
+    @Override
+    public void goForwardClicked(View v) {
```

Comment:
@nategbrown9 last thing, I added the camera permission request when you tap "next" on the first screen of each CRF task to match iOS flow and copy.